trex-scripts/tests/priority_scheduling.py

```diff
@@ -0,0 +1,152 @@
+# SPDX-FileCopyrightText: Copyright 2020-present Open Networking Foundation.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import os
+import json
+from argparse import ArgumentParser
+from datetime import datetime
+from scapy.layers.all import IP, TCP, UDP, Ether
+
+from lib.base_test import StatelessTest
+from lib.utils import list_port_status,get_readable_port_stats
+from lib.xnt import analysis_report_pcap
+from trex_stl_lib.api import STLPktBuilder, STLStream, STLTXCont, STLTXSingleBurst, STLFlowLatencyStats
+
+
+SOURCE_MAC = "00:00:00:00:00:01"
+DEST_MAC   = "00:00:00:00:00:02"
+
+SENDER_PORT   = [0]
+RECEIVER_PORT = [1]
+
+# In this test case, we consider two types of flows: 1) delay critical, and 2) best effort
+# assume max. latency for delay critical flows
+LATENCY_DC_MAX_uSEC = 1000   # in mircoseconds
```

Comment:
nit: the convention for constants is all caps
```suggestion
LATENCY_DC_MAX_USEC = 1000   # in microseconds
```